### DISCUSSION TITLE
Usage on completions

### DISCUSSION THREAD

**Discussion by drksndrs** [MacPaw/OpenAI#35] [Ideas]
Can you guys add to the CompletionsResult struct so we can access how many tokens were used?

**Comment by Krivoblotsky** [comment]
Hey, @drksndrs. 
Sure, I will add it to the next release. Thanks for the suggestion 🔥

**Comment by Krivoblotsky** [comment]
A corresponding issue has been [created](https://github.com/MacPaw/OpenAI/issues/36).

**Comment by drksndrs** [comment]
Awesome! Amazing repository btw.
